doc/classes/ProjectSettings.xml

```diff
@@ -444,21 +444,21 @@
 		<member name="display/window/size/fullscreen" type="bool" setter="" getter="" default="false">
 			Sets the window to full screen when it starts.
 		</member>
-		<member name="display/window/size/height" type="int" setter="" getter="" default="600">
+		<member name="display/window/size/render_height" type="int" setter="" getter="" default="600">
 			Sets the game's main viewport height. On desktop platforms, this is the default window size. Stretch mode settings also use this as a reference when enabled.
 		</member>
+		<member name="display/window/size/render_width" type="int" setter="" getter="" default="1024">
+			Sets the game's main viewport width. On desktop platforms, this is the default window size. Stretch mode settings also use this as a reference when enabled.
+		</member>
 		<member name="display/window/size/resizable" type="bool" setter="" getter="" default="true">
 			Allows the window to be resizable by default.
 		</member>
-		<member name="display/window/size/test_height" type="int" setter="" getter="" default="0">
+		<member name="display/window/size/window_height" type="int" setter="" getter="" default="0">
 			If greater than zero, overrides the window height when running the game. Useful for testing stretch modes.
 		</member>
-		<member name="display/window/size/test_width" type="int" setter="" getter="" default="0">
+		<member name="display/window/size/window_width" type="int" setter="" getter="" default="0">
 			If greater than zero, overrides the window width when running the game. Useful for testing stretch modes.
```

Comment:
"If greater than zero, overrides the **render width** when running the game." + same for height.

Reply:
Are you sure? There is already `render_width` and it's different than `window_width`. Maybe you mean `render_width` instead?

Reply:
Oh, you're right, it makes sense as-is, but it could be better.

I was thinking in terms of `window_width` overriding the property called `render_width`. Perhaps there should be some reference to `render_width`? Perhaps:

"If greater than zero, overrides `render_width` for the window width when running the game."

Reply:
How about this: `"If greater than zero, overrides the window width and [code]display/window/size/render_width[/code] of running game. Useful for testing stretch modes."`

Reply:
@Chaosus You can link to members directly like this: `[member display/window/size/render_width]`

Reply:
I knew about it, I just wanted to see a nice tooltip for the property in settings because currently, you see [member] tag which is not nice. But thanks for the notice I will fix it as you said.